doc/dev/custom_devices.rst

```diff
@@ -149,11 +153,14 @@ After doing so, Catalyst should be able to interface with your custom device.
 
         @staticmethod
         def get_c_interface():
-            """Location to shared object with C/C++ implementation"""
-            return "full/path/to/libdummy_device.so"
+            """ The tuple of davice name and the location
+                to shared object with C/C++ implementation.
+            """
+
+            return "CustomDevice", "full/path/to/libdummy_device.so"
```

Comment:
```suggestion
            """Returns a tuple consisting of the device name, and
            the location to the shared object with the C/C++ device implementation.
            """

            return "CustomDevice", "full/path/to/libdummy_device.so"
```

Reply:
@maliasadi it's not entirely clear to me if the path here is relative to some directory?

Reply:
Should be absolute. Maybe "absolute/path/to/libdummy_device.so" would be clearer than "full/path/to/libdummy_device.so"?

Reply:
I think mentioning this in the text might be helpful?

Reply:
yep, agree! :100: